setup.py

```diff
@@ -66,6 +66,9 @@
             # type stubs for mypy
             "types-PyYAML==6.0.10",
             "types-tabulate==0.9.0.1",
+            "termcolor==2.2.0",
+            "colorama==0.4.4",
```

Comment:
these should remain in `requirements` while the `types-*` dependencies should stay here. type checking is only done during development/CI, that's why they're separated out. dependencies that are needed for FLOSS to run should be listed in `requirements`